startups/serializers.py

```diff
@@ -60,21 +61,14 @@ def validate_location(self, data):
         return data
 
     def validate_contact_phone(self, data):
-        if not re.match(r'^\d{3}-\d{3}-\d{4}$', data):
+        if not re.match(PHONE_NUMBER_REGEX, data):
             raise serializers.ValidationError("Mobile phone number must be in the format XXX-XXX-XXXX")
         return data
 
-    def validate_contact_email(self, data):
-        regex_for_email = r'^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
-        # Check if the email match pattern
-        if not re.match(regex_for_email, data):
-            raise serializers.ValidationError({"Error": "Invalid email address"})
-        return data
-
```

Comment:
email must contain "@"

Reply:
We removed email validation because it works the same as model.EmailField validation in django, wait a little bit till my new commit wth all issues fixed

Reply:
👍🏻